.changeset/nice-eyes-flash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@cloudflare/next-on-pages': minor
+---
+
+Added support to utilize the package manager used in the project
```

Comment:
```suggestion
Added support to utilize the package manager used in the project for local development
```
(to reflect what I mentioned here: https://github.com/cloudflare/next-on-pages/pull/145#issuecomment-1497161287)